### ISSUE TITLE
[Checkout Shipping Options] Order summary Shipping updates

### ISSUE THREAD

**Issue by opr** [woothemes/woocommerce#50557]
This is a followup to https://github.com/woocommerce/woocommerce/issues/46948

```[tasklist]
- [ ] When no address has been provided and there are no shipping options, change the wording `No shipping options available` in the order summary section to `Enter address to calculate`, and display it in 80% of the font size of the `Shipping` label and normal, neither `<strong>` nor `<em>`. (See image 1 for expected result)
- [ ] When an address _has_ been provided, change the wording `No shipping options available` in the order summary section to `No available delivery option`, and display it in 80% of the font size of the `Shipping` label and normal, neither `<strong>` nor `<em>`. (See image 2 for expected result)
```

### Image 1
<img width="411" alt="image" src="https://github.com/user-attachments/assets/70d1cd0f-ce82-46e7-91f4-24bb963d8f1d">

### Image 2
<img width="408" alt="image" src="https://github.com/user-attachments/assets/cc560975-9fda-482a-a769-7c9cf9a1716e">

**Comment by ethanclevenger91** [comment]
@opr @samueljseay was this ever implemented for the Cart block? We're still seeing the old messaging and opened a ticket about it on the WP.org support forum. We were told to make a translation, but clearly someone else noticed that it wasn't a good UX. See https://wordpress.org/support/topic/no-delivery-options-available-for-united-states-us-in-cart/

**Comment by opr** [comment]
Hey @ethanclevenger91 I had a look through your thread on the forum. From what I understand, the shopper's `shipping_country` is assigned automatically through geolocation, and then the label in the cart shipping calculator is incorrect?

I just tested this locally (I didn't geolocate my customer, but I did automatically set their location to a zone with no shipping options, and also tried one _with_ options), this is my result:

<img width="454" alt="Image" src="https://github.com/user-attachments/assets/46a013be-e556-41c6-9536-44757141315d" />

Have you tried with the latest version of WooCommerce 9.8.2? Does the issue persist there?
